content/S2016/S16-event-teaparty.md

```diff
@@ -0,0 +1,24 @@
+title: Tea Party and Volunteer Meeting
+Date: 2016-4-27
+Category: Events
+Tags: social, volunteer, big csters
+Slug: tea-party-s16
+Author: Evy Kassirer
+Summary: Join us for tea and for information on volunteering with us!
```

Comment:
I personally think this wording is a bit more clear, but no strong feelings either way